Shell/Options.cpp

```diff
@@ -721,6 +721,21 @@ void Options::init()
     _lookup.insert(&_showSimplOrdering);
     _showSimplOrdering.tag(OptionTag::OUTPUT);
 
+#if VAMPIRE_CLAUSE_TRACING
+
+    _traceBackward = IntOptionValue("trace_bwd","",0);
+    _traceBackward.description = "The id of a clause you want to see all predecesors (unites used to derive the clause).";
+    _lookup.insert(&_traceBackward);
+    _traceBackward.tag(OptionTag::OUTPUT);
+
+    _traceForward = IntOptionValue("trace_fwd","",-1);
```

Comment:
There was this TODO "Make UnsignedOptionValue", can we have it? (Or is `-1` important here?)

Reply:
Yess indeed. So the thing is that for making an `OptionalOptionValue<T>` I'd need to refactor the whole options thing a little bit. (see branch `option-refactor`), and I thought I'd rather keep this out of here (for now). 
I thought I'd rather make a later PR maybe that implements this. With the refactoring from that branch we could also create things lik `OptionValue<List<T>>, OptionValue<pair<A,B>>` and ` and so on. I think it would be nice to do that as we could use that to tidy up a lot of specialized option value types. But I thought it'd b better to do that in a separate PR to keep this one smaller.